src/components/TaskStatusBadge/index.tsx

```diff
@@ -31,6 +31,10 @@ interface TaskStatusBadgeProps {
   status: string;
 }
 const TaskStatusBadge: React.VFC<TaskStatusBadgeProps> = ({ status }) => {
+  if (!status) {
+    return <></>;
+  }
+
   let displayStatus = getStatusBadgeCopy(status);
 
   if (taskStatusToCopy[status] === undefined) {
```

Comment:
We might be able to simplify `L34 ~ L46` into:

```
  if (!status || !taskStatusToCopy[status]) {
    const err = new Error(`Status '${status}' is not a valid task status`);
    reportError(err).warning();
    return null;
  }

  const displayStatus = getStatusBadgeCopy(taskStatusToCopy[status]);
```

Reply:
I'll like to display the status even if it's not official